bin/play.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../lib/tictactoe'
+require_relative '../lib/main_io'
+require_relative '../lib/board'
+require_relative '../lib/presenter'
+require_relative '../lib/move_validator'
+require_relative '../lib/game'
+
+board = Board.new(layout: [*1..9].map!(&:to_s))
+main_io = MainIO.new(stdin: $stdin, stdout: $stdout)
```

Comment:
It looks like you initialize the `board` and `main_io` in `main.rb` too. Can you just do it in one place?

Reply:
I'm not sure if you want to pass in layout for the board class. This board class uses an array to represent the board structure which all the methods in this class operate on. This layout argument sends a wrong signal that it'll work the same way if I choose to use a different data structure.